flowermd/base/system.py

```diff
@@ -631,6 +631,9 @@ class Pack(System):
         Minimum separation (nm) between particles of different molecules.
     seed : int, default 12345
         Change seed to be passed to PACKMOL for different starting positions
+    unique_molecules : bool, default True
+        Change to False to assume each compound to be initialized has a
```

Comment:
Shouldn't this be the opposite. If all the molecules have unique configurations, then this should be True. We should say something like "If `False`  then assume each compound to be initialized in the system has identical configuration and chemical identity. Setting this to `False` can help improve packing performance for large systems without unique starting molecules."